tasks/users.yml

```diff
@@ -12,6 +12,15 @@
     interface: "{{ os_projects_interface | default(omit, true) }}"
     domain: "{{ project.user_domain }}"
   environment: "{{ os_projects_environment }}"
+  register: os_user_facts_result
+
+- name: Ensure openstack_users is present
+  # Workaround for os_user_facts not setting the openstack_users fact, See:
+  # https://github.com/stackhpc/ansible-role-os-projects/issues/29
```

Comment:
It's due to a change in how these modules work. They have been renamed from os_user_facts to os_user_info. Since Ansible 2.10 they no longer register facts, but return a variable in the result.

Reply:
Does the fix seem reasonable? I think we retain backwards compatibility this way. Or do you see a better solution?

Reply:
We should be explicitly specifying which Ansible versions are supported by the role, possibly also bump `min_ansible_version` in role's metadata.

Reply:
Doesn't `min_ansible_version` imply which versions are supported? We could bump to >=2.10 and get rid of this compatibility task, but I'm not sure everyone would want to jump to that version just yet.